utensor_cgen/backend/graph_lower/generic_graph_lower.py

```diff
@@ -37,22 +37,22 @@ def apply(self, ugraph):
     life_span = defaultdict(lambda: [None, None])
     for op_info in ugraph.ops_info.values():
       for tensor in op_info.input_tensors:
-        ref_cnts[tensor.name] += 1
+        ref_cnts[tensor] += 1
```

Comment:
How do you handle inputs to the model? Pretty sure the logic will increment these but never decrement, but I just want to double check

Reply:
Oh, I didn't deal with inputs in this case.
Because the model inputs is declared by the users, the users should be responsible for its creation and destruction.
And I do decrement the ref cnt so I can find the life span of the tensors:
https://github.com/uTensor/utensor_cgen/pull/110/files/5d030fb3ea9ada6b73d16f7f2682c3a3c872fcd4#diff-ad6830d2f958b649ebbb340c55ab59e8R46